### DISCUSSION TITLE
Future of this project

### DISCUSSION THREAD

**Discussion by bobcate** [anapnoe/stable-diffusion-webui-ux#229] [General]
[SD.Next](https://github.com/vladmandic/automatic/) released with a [modern UI](https://github.com/BinaryQuantumSoul/sdnext-modernui). I hope that doesn't mean this project will not be integrated with SD.Next.
I've tried that modern UI and I hate it very much. Yours is 10x better than that, imo.
The last update you've pushed was 2 months ago, that's worrisome. Please don't abandon this project.

**Comment by anapnoe** [comment]
Thank you @bobcate I will take some time this weekend to update the project. Due to time constraints I won't be able to do any work for SD.Next.

**Comment by GreenGhost21** [comment]
Any progress updates? I far prefer this UI over a1111's default UI, so it would be amazing to see this updated.

**Reply by anapnoe** [reply]
Thank you the project has been updated :)
